src/javascript/app/pages/user/account/top_up_virtual/top_up_virtual.js

```diff
@@ -20,10 +18,7 @@ const TopUpVirtual = (() => {
                 showMessage(response.error.message, false);
             } else {
                 showMessage(
-                    localize('[_1] has been credited into your Virtual Account: [_2].', [
-                        formatMoney(response.topup_virtual.currency, response.topup_virtual.amount),
-                        Client.get('loginid'),
-                    ]),
+                    localize('Your virtual balance has been reset.'),
```

Comment:
should we mention it's been reset to 10,000.00 USD?

Reply:
This is the approved copy, you may have a look at this issue/card https://redmine.deriv.cloud/issues/16644